Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -92,6 +91,66 @@ namespace RDKit {
       return true;
     }
 
+    // the minimization using experimental torsion angle preferences
+    void _minimizeWithExpTorsions(RDGeom::PointPtrVect &positions,
+                                  DistGeom::BoundsMatPtr mmat,
+                                  double optimizerForceTol, double basinThresh,
+                                  std::vector<std::pair<int, int> > bonds,
```

Comment:
lots of vectors being copied here. These vector arguments should probably all be references. You also should think about which of them are `const` (maybe none, but still to be thought about).